src/applications/pre-need/utils/helpers.js

```diff
@@ -341,6 +341,9 @@ export const serviceRecordsUI = {
   },
   items: {
     'ui:order': ['serviceBranch', '*'],
+    'ui:options': {
+      ariaLabelForEditButtonOnReview: 'Service Period',
```

Comment:
The original [issue](https://github.com/usds/us-forms-system/issues/320#issuecomment-449067162) suggested we add an aria-tag to the edit button on the array list items. 

ArrayList items don't have access to the top level `ui:options.itemName`. They do support using this attribute when it is set.

Reply:
this may not be a tenable approach to solving this issue- there's not an easy way to just search the code base to find all of places an object is being used in an array list.